alvr/packets/src/lib.rs

```diff
@@ -119,6 +119,7 @@ pub struct NegotiatedStreamingConfig {
     // This is needed to detect when to use SteamVR hand trackers. This does NOT imply if multimodal
     // input is supported
     pub use_multimodal_protocol: bool,
+    pub wired: Option<bool>,
```

Comment:
Sorry, I was a bit confused when I explained it, this can be a simple bool, not Option<bool>